nucypher/policy/conditions/lingo.py

```diff
@@ -62,11 +70,12 @@ def __init__(self, comparator: str, value: Union[int, str]):
     def sanitize(self, comparator: str, value: str) -> Tuple[str, str]:
```

Comment:
We should adjust the typehint here. If `value` is already a `str` then we shouldn't need to do `str(value)` on L73. 
Seems like our options are:
1. Not have a typehint for value
OR
2. Have the typehint for value match what is in the constructor i.e. `Union[int, str]` 
OR
3. We pass `str(value)` to `sanitize(...)` on L66. 

wdyt?

Side note: should we change `sanitize(...)` ->  `_sanitize(...)` since it is only used by the constructor? Is there any expected usage outside of that?